### DISCUSSION TITLE
建议：使用网站url记录页面的状态

### DISCUSSION THREAD

**Discussion by luwenhegithub** [Gamuxorg/bbs#340] [建议和闲聊]
建议使用网站url记录页面的状态

比如在游戏列表的页面，记录选中的游戏类型和翻页翻到第几页了

这样当页面刷新和退回的时候，能大大提高操作效率

**Comment by shouhuanxiaoji** [comment]
是的，有计划支持，目前搜索页已经实现了url参数，列表页还在想办法。

**Comment by shouhuanxiaoji** [comment]
列表页已经支持url参数
